### DISCUSSION TITLE
SHA1 title in XPath Web scraping

### DISCUSSION THREAD

**Discussion by god81000** [FreshRSS/FreshRSS#6154] [Q&A] [answered]
Hello, 
I use FreshRSS and its HTML/XPath scraping system to scrape articles, but when I try to retrieve articles from 'https://www.proofpoint.com/us/blog/threat-insight', for example, the title is displayed in SHA1 (Image). Here is the configuration: Does anyone have a solution to this issue?

![image](https://github.com/FreshRSS/FreshRSS/assets/75736124/96a1a396-5d8f-49fd-afe2-18263c40cdc6)
![image](https://github.com/FreshRSS/FreshRSS/assets/75736124/0f1ef275-a31f-425c-815c-b0f1554aca32)

**Comment by Alkarex** [comment, marked as answer]
That is a sign that your title selector does not work (returns a blank result).
Your XPath should also be simplified.

It works for me with:
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/0c582a05-7b1f-4202-b278-639192dc0159)

Or import the following OPML:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<opml xmlns:frss="https://freshrss.org/opml" version="2.0">
  <head>
    <title>FreshRSS</title>
    <dateCreated>Mon, 04 Mar 2024 22:51:37 +0100</dateCreated>
  </head>
  <body>
      <outline text="Threat Insight Information &amp; Resources | Proofpoint Blog US" type="HTML+XPath"
        xmlUrl="https://www.proofpoint.com/us/blog/threat-insight"
        htmlUrl="https://www.proofpoint.com/us/blog/threat-insight"
        description="Flux RSS de Threat Insight Information &amp; Resources | Proofpoint Blog US"
        frss:xPathItem="//div[@class=&quot;blog-mosaic&quot;]//div[@role=&quot;article&quot;]"
        frss:xPathItemTitle="descendant::div[@class=&quot;blog-mosaic__title&quot;]"
        frss:xPathItemContent="."
        frss:xPathItemUri="descendant::a/@href"/>
  </body>
</opml>
```

**Reply by god81000** [reply]
Thank you for your complete answer, it now works on my side, thank you.
